tests/src/application/controllers/forget-password.spec.ts

```diff
@@ -95,13 +98,20 @@ describe('ForgetPassword Controller', () => {
     jest.spyOn(emailValidatorStub, 'isValid').mockImplementationOnce(() => {
       throw new Error()
     })
-    const httpRequest = {
-      body: {
-        email: 'any_email@mail.com'
-      }
-    }
-
-    const httpResponse = await sut.handle(httpRequest)
+    const httpResponse = await sut.handle(makeFakeRequest())
     expect(httpResponse).toEqual(makeFakeServerError())
   })
+
+  it('Should return 400 if no guardian is found with the provided email', async () => {
```

Comment:
We refactor `expect(httpResponse).toEqual(badRequest(new NotFoundError('email')))` to simplify that logic of `expect(httpResponse.statusCode).toBe(400)`, maybe make sense change tests messages to "should return badRequest" otherwise "should return 400".

Reply:
I think it is better to keep it as it is, because the test is to check the http status that is returned and not to check if we use badRequest